stripe/_api_requestor.py

```diff
@@ -266,7 +276,7 @@ def request_stream(
         )
         return resp
 
-    async def request_stream_async(
+    async def _request_stream_async(
```

Comment:
nit method naming consistency: Was there a specific reason to prefix this with an underscore? `APIRequestor` is not publically exposed now so every method is "private", and the other `request_...` methods in the class are not prefixed.

Reply:
No reason, just sloppiness/forgetfulness from me. Will revert.

Reply:
Fixed in [Revert unnecessary underscore](https://github.com/stripe/stripe-python/pull/1233/commits/f02f30fe6f2fd70a593e1369a201e34b451e431c)